src/lib/utils.js

```diff
@@ -1,3 +1,5 @@
+import isEmpty from 'lodash.isempty';
```

Comment:
@milushov is this necessary? wouldn't `snapshot.val()` value always only return null if the value was empty?